site/packages/aa-alchemy/provider/withAlchemyEnhancedApis.md

```diff
@@ -0,0 +1,52 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: AlchemyProvider • withAlchemyEnhancedApis
+  - - meta
+    - name: description
+      content: Overview of the withAlchemyEnhancedApis method on Alchemy Provider in aa-alchemy
+  - - meta
+    - property: og:description
+      content: Overview of the withAlchemyEnhancedApis method on Alchemy Provider in aa-alchemy
+---
+
+# withAlchemyEnhancedApis
+
+`withAlchemyEnhancedApis` is a method on `AlchemyProvider` that you can optionally call to create a new provider instance with added methods that access the Alchemy [Enhanced APIs](https://www.alchemy.com/enhanced-apis) via the [Alchemy SDK](https://github.com/alchemyplatform/alchemy-sdk-js).
+
+:::tip Note
+This method requires a peer dependency on the [`alchemy-sdk`](https://github.com/alchemyplatform/alchemy-sdk-js) package, as the input to this method is an Alchemy SDK client.
+
+The Alchemy SDK client must be configured with the same API key and network as the AlchemyProvider. This method validates such at runtime.
+
+Additionally, since the Alchemy SDK client does not yet support JWT authentication, an `AlchemyProvider` initialized with JWTs cannot use this method. They must be initialized with an API key or RPC URL.
+:::
```

Comment:
what if we just initialize the Alchemy instance internally for them?

Would that simplify the dev ex?

Reply:
High level this is about peer dependency vs. dependency for alchemy-sdk-js. 

Still think peer dependency is the right move to avoid package bloat. Additionally, passing in the alchemy object direct is a better application of dependency injection (which I prefer).